IOSAccessAssessment/Services/ChangesetService.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ChangesetService.swift
+//  IOSAccessAssessment
+//
+//  Created by Mariana Piz on 11.11.2024.
+//
+
+import Foundation
+
+struct NodeData {
+    let latitude: Double
+    let longitude: Double
+}
+
+class ChangesetService {
+    
+    private enum Constants {
+        static let baseUrl = "https://osm.workspaces-stage.sidewalks.washington.edu/api/0.6"
+        static let workspaceId = "168"
+    }
+    
+    static let shared = ChangesetService()
+    private init() {}
+    
+    private let accessToken = KeychainService().getValue(for: .accessToken)
+    private(set) var changesetId: String?
+    
+    func openChangeset(completion: @escaping (Result<String, Error>) -> Void) {
+        guard let url = URL(string: "\(Constants.baseUrl)/changeset/create"),
```

Comment:
So the fact that we now have two Constants structs will lead to confusion.
No problem tho, we will move all this to an ideal place in a future refactor.

Reply:
This is not a problem because the Constants enum is private for each struct